test/sconsign/script/Configure.py

```diff
@@ -84,10 +84,10 @@
         %(sig_re)s \[.*\]
 conftest_%(sig_re)s_0_%(sig_re)s%(_obj)s:
         %(_sconf_temp_conftest_0_c)s: %(sig_re)s \d+ \d+
-        %(CC)s: %(sig_re)s \d+ \d+
+        %(CC)s: %(sig_re)s None None
```

Comment:
this is the compiler which is considered a non-conftest node, so if only configure checks are run, then the node info is not stored.